port.h

```diff
@@ -90,4 +90,19 @@ SOFTWARE.
 #error "Unknown architecture."
 #endif
 
+#ifdef BRICKS_WINDOWS
+
+// #include <windows.h>  -- `#include <windows.h>` does not play well with WSA -- D.K.
+
+// These two headers seem to work fine for both WSA and `FindFirst*` files.
+#include <WS2tcpip.h>
+#include <corecrt_io.h>
+
+// Visual Studio does not define `M_PI`.
```

Comment:
Should a compiler define a math constant? I thought some math-related headers should define it.